README.md

```diff
@@ -291,65 +290,70 @@ and it is far easier to work with them when you have deeper understanding yourse
+fundamentals about how companies are formed and owned.
+What are stock and shares?
+And how are shares in a company created and owned, and how does that change over time?
 
-This requires discussing starting with the basics of how companies and stock work in the
-United States.
-If you think you know all this, you can [jump ahead](#equity-compensation-basics) to
-compensation.
+Those familiar with stock, stock corporations, and how startups take investment may
+[jump ahead](#equity-compensation-basics) to compensation.
 
 ### What is a company?
 
+*Company*, *corporation*, and *startup* are words often heard but it’s worth clarifying the
+differences.
+
 - 🄳 A **company** is a legal entity formed under corporate law for the purpose of conducting
   trade. In the United States,
   [several kinds of business entities](https://en.wikipedia.org/wiki/Types_of_business_entity#United_States)
-  may be formed, or [**incorporated**](https://en.wikipedia.org/wiki/Incorporation_(business)),
-  under the laws of a specific state.
-- 🄳 A [**startup**](https://en.wikipedia.org/wiki/Startup_company) is an emerging company that
-  aspires to grow quickly.
-  Once a company is established in the market and successful for a while, it usually stops
-  being called a startup.
-  - ☝️ The term *startup* is pretty informal.
-    It’s not a legal concept, and not everyone uses it consistently.
-    For our purposes, it’s best just to think of it as the young period of a fast-growing
-    company.
-  - 🚧 Groundwork:
-    What are good stats on how many people work in startups vs established companies?
-- Companies in the US include sole proprietorships, partnerships, limited liability
-  companies (LLCs), S corporations, and C corporations.
-  - 🔸 This Guide does not cover equity compensation in LLCs, as they are rarely used as the
-    choice of entity for technology startups, and equity compensation in LLCs is dramatically
-    different from equity compensation in corporations.
-    So we’ll focus on the one type of company most readers of this Guide will be concerned
-    with: The C corporation.
-- 🄳 A [**C corporation (or C corp)**](https://en.wikipedia.org/wiki/C_corporation) is a type of
-  business entity in the United States.
+  are common, including sole proprietorships, partnerships, limited liability companies
+  (LLCs), S corporations, and C corporations.
+  Several of these kinds are called **corporations** and they are formed, or
+  [**incorporated**](https://en.wikipedia.org/wiki/Incorporation_(business)), under the laws of a
+  specific state.
+- 🔸 This Guide [does not cover](#scope) equity compensation in LLCs, S corporations,
+  partnerships, or sole proprietorships.
+  Equity compensation in each of these kinds of business entities is significantly different
+  than in C corporations.
+- 🄳 A [**C corporation (or C corp)**](https://en.wikipedia.org/wiki/C_corporation) is one type of
```

Comment:
It's less clear now why we only talk about C corps. What does "several of these kinds" mean? Company v. corporation not clear. Also, "equity component" is more correct for partnerships and sole proprietorships and S corps, as it's not compensation per se in companies of one or two people, but rather an input of capital from personal accounts for the success of the company, with the return on investment not being calculated by anything other than a balance sheet. Consider reverting to (still open to further clarification on company v. corp): 

"-Companies in the US include sole proprietorships, partnerships, limited liability companies (LLCs), S corporations, and C corporations.
-🔸 This Guide does not cover equity compensation in LLCs, as they are rarely used as the choice of entity for startups in technology and in non-tech sectors, and equity compensation in LLCs is dramatically different from equity compensation in corporations. The equity components of sole proprietorships, partnerships, and S corps are equally divergent from equity compensation in typical public and private companies, including startups, which are all overwhelmingly C corporations.
-🄳 A C corporation (or C corp) is a type of business entity in the United States. Specifically, it’s a kind of stock corporation (or joint-stock company) with certain federal tax treatment."

Reply:
OK. I still am hesitant about bringing up tech, non-tech, startups, and private/public here. I'm rewording and moving the caution below.